### PR TITLE
Write first draft of chapter on pip installing.

### LINKED ISSUE #112: Write section/chapter on conda/pip installing library
Use `matching` as an example.

Question: 

- Does this go in the command line chapter or in it's own chapter?

I'm leaning towards its own chapter.

### PR DESCRIPTION
**Pull request by drvinceknight**
Closes #112

### CODE REVIEW

File: book/further-information/04-pip-installing/tutorial/main.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+game = StableMarriage(pilots, copilots)
+game.solve()
+```
+
+As discussed, this particular pairing will pair any pilots or co-pilots that
+would rather be paired with someone who would rather be paired with them.
+
+The `matching` documentation has an excellent set of documentation that includes
+a theoretic overview of the mathematics used.
```
Comment (daffidwilde) [inline book/further-information/04-pip-installing/tutorial/main.md:190]: I'm not sure if the double use of "documentation" here reads smoothly

File: book/further-information/04-pip-installing/how/main.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+````
+
+For example to install `ciw` a library used to study queuing systems:
+
+```
+$ python -m pip install ciw
+```
+
+```{attention}
+Well documented libraries will always have installation installation. It is
```
Comment (daffidwilde) [inline book/further-information/04-pip-installing/how/main.md:35]:
```suggestion
Well documented libraries will always have installation instructions. It is
```

File: book/further-information/04-pip-installing/why/main.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Further information.
+
+## Why do we do `python -m pip install` and not `pip install`
+
+In some places you will see that to install a python library you can run the
```
Comment (daffidwilde) [inline book/further-information/04-pip-installing/why/main.md:19]:
```suggestion
In some places you will see that to install a Python library you can run the
```

File: book/further-information/04-pip-installing/why/main.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+jupyter-book
+jupytext
+nbval
+proselint
+pytest
+```
+
+## What is the Python package index
+
+The Python package index commonly referred to as PyPi is:
```
Comment (daffidwilde) [inline book/further-information/04-pip-installing/why/main.md:85]:
```suggestion
The Python package index commonly referred to as PyPI is:
```

Is that right?

Comment (daffidwilde) [reply book/further-information/04-pip-installing/why/main.md:85]: You use `PyPI` in line 87 below.

**Review by daffidwilde — COMMENTED** [review]
Looks like a good edition to the book 😄 I've just caught a couple of minor typos, but it's a nice tutorial for `matching`.

File: book/further-information/04-pip-installing/why/main.md (hunk 4 above)
Comment (drvinceknight) [reply book/further-information/04-pip-installing/why/main.md]: I'm so inconsistent all over the place. At some point I need to go back through everything for that.

File: book/further-information/04-pip-installing/tutorial/main.md (hunk 1 above)
Comment (drvinceknight) [reply book/further-information/04-pip-installing/tutorial/main.md]: Yup, this is dumb.

**Comment by drvinceknight** [comment]
Tests are failing due to what looks like an update of sympy (this is why those tests are here):

- [ ] Update sympy locally
- [ ] Rebuild all notebooks

**Pull request merged**